pkg/daemon/controller/controller.go

```diff
@@ -258,13 +275,25 @@ func (sc *ServiceController) Reconcile(_ context.Context, req ctrl.Request) (res
 	if err = sc.waitUntilNodeReady(); err != nil {
 		return ctrl.Result{}, fmt.Errorf("error waiting for node to become ready")
 	}
-	// Version annotation is the indicator that the node was fully configured by this version of the services ConfigMap
-	if err = metadata.ApplyVersionAnnotation(sc.ctx, sc.client, node, desiredVersion); err != nil {
-		return ctrl.Result{}, fmt.Errorf("error updating version annotation on node %s: %w", sc.nodeName, err)
+	if err = sc.markNodeConfigured(&node, desiredVersion); err != nil {
+		return ctrl.Result{}, err
 	}
 	return ctrl.Result{}, nil
 }
 
+// markNodeConfigured marks the node as configured and schedulable
+func (sc *ServiceController) markNodeConfigured(node *core.Node, desiredVersion string) error {
+	err := drain.RunCordonOrUncordon(sc.drainer, node, false)
```

Comment:
I wish there were helpers for Cordon and Uncordon. Please add a comment here saying you are uncordoning.